### ISSUE TITLE
No syntax highlightning in VS2015 Enterprise

### ISSUE THREAD

**Issue by RobDesideri** [madskristensen/TextmateBundleInstaller#88]
### Installed product versions
- Visual Studio: 2015 Enterprise
- This extension: 1.8.61

### Description
Extension not run in my environement. I have tried a .go file. a .lisp file, a .yml file but no syntax highlighting appears.

### Steps to recreate
1. New empty solution in VS
2. Add elements to solution
3. Add test.go as file
4. Open test.go and write little hello word function

### Current behavior
Code visualized as plain text.

### Expected behavior
Syntax highlightning.

**Comment by madskristensen** [comment]
Do you have VS 2015 Update 3 installed?

**Comment by RobDesideri** [comment]
@madskristensen Yes, it is the 14.0.25431.01 Update 3 version.

**Comment by madskristensen** [comment]
Perhaps it's ReSharper interfering. If you have it installed, will it work if you disable it?

**Comment by RobDesideri** [comment]
@madskristensen no, I haven't installed ReSharper.

**Comment by madskristensen** [comment]
Can you check that there is a bunch of folders inside `%userprofile%\.vs\Extensions`?

**Comment by RobDesideri** [comment]
@madskristensen yes, there are folders for each language.

**Comment by jaxuk** [comment]
I've got the same issue on VS2015 Enterprise

**Comment by PeterMacej** [comment]
I had the same problem. Are you using any custom Visual Studio template? The 'Visual Studio Extension for TextMate Grammars' seems to work with only the basic VS templates (light, dark, blue). See https://visualstudio.uservoice.com/forums/121579-visual-studio-ide/suggestions/15880407-add-theme-support-for-textmate-syntax-extensions
and https://connect.microsoft.com/VisualStudio/Feedback/Details/2343983
I've found a workaround and posted it at the second link, in " Workarounds" section.

**Comment by jaxuk** [comment]
I'm using the standard "dark" theme. I've tried changing it back to "light" (the default) but the highlighting still doesn't appear.

**Comment by RobDesideri** [comment]
I'm using the standard theme @PeterMacej

**Comment by markhurd** [comment]
I have two installations of VS2015 Update 3 (a 64-bit desktop and a 32-bit laptop, both Windows 10 using the standard "dark" theme) that fail to show syntax highlighting for .cmd and .lisp files. On the same systems, I've installed VS2017RC and the syntax highlighting and vertical dashed lines are working on my Windows 10 64-bit machine.

**Comment by madskristensen** [comment]
@markhurd Can you check that there is a bunch of folders inside %userprofile%\.vs\Extensions?

**Comment by markhurd** [comment]
No, but %userprofile%**\**.vs\Extensions has folders on both systems,

**Comment by markhurd** [comment]
I've now had syntax highlighting appear in VS2017RC on the 32-bit laptop. I have no idea what caused it to start working. I haven't yet restarted to confirm I'll lose it again, and, in differences panels, it's only working for the old file, not the current one.

**Comment by nanoant** [comment]
For sake of others looking for the "workaround" - https://connect.microsoft.com/VisualStudio/Feedback/Details/2343983 link provided by @PeterMacej no longer works (connect website has been shut down). The "workaround" is to modify:

1. `c:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\Extensions\XXXXXXXX.YYY\Starterkit\Themes\theme.metadata`
    where `XXXXX.YYY` is some (random) auto generated identifier, and add the lowercase name of your theme into `vs-themes` and `default`.

2. Make sure you select the latest (newest) version of the extension with `TMLanguage.pkgdef` file in its root folder not `TMLanguage.pkgdef.deleteme`, which is there in the old (inactive) extensions.

P.S. I have lost couple of hours to find this workaround, all because "connect" site was shut down in favor new site, but the tickets were not migrated, essentially making all archives not accessible anymore. 😢
